tests/testthat/test-skip.R

```diff
@@ -25,6 +25,7 @@ test_that("Skip env vars", {
   expect_skip_with_env(c("APPVEYOR" = "True"), skip_on_appveyor)
   expect_skip_with_env(c("R_COVR" = "true"), skip_on_covr)
   expect_skip_with_env(c("BBS_HOME" = "asdf"), skip_on_bioc)
+  expect_skip_with_env(c("GITHUB_ACTIONS" = "asdf"), skip_on_gh_actions)
```

Comment:
I think this needs to be "true", or else the test will fail. In the skip you are checking the value of the environment variable is `"true"` This is why it is currently failing on the CI.